tests/MongoDB.Driver.Core.Tests/Specifications/server-discovery-and-monitoring/TestRunner.cs

```diff
@@ -282,38 +271,26 @@ private ICluster BuildCluster(BsonDocument definition)
                 .CreateCluster();
         }
 
-        private class TestCaseFactory : IEnumerable<object[]>
+        private class TestCaseFactory : JsonDrivenTestCaseFactory
         {
-            public IEnumerator<object[]> GetEnumerator()
+            #region static
+            private static readonly string[] __ignoredTestNames =
```

Comment:
I think this variable should be named `__ignoredTestFolders`.

Also, since there is only one consider not using an array.

Reply:
done